### PR TITLE
fix: update README on new testing process

### PR DESCRIPTION
**Pull request by lucabello**
## Issue
We probably don't need to check that `rockcraft` itself is working correctly (i.e., that it's correctly installing apt packages).

## Solution
Remove the `goss` check on the presence of the `ca-certificates` package.

### CODE REVIEW

**Comment by dstathis** [comment]
Is this check causing problems?

File: goss.yaml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 process:
   mimir:
     running: true
-package:
-  ca-certificates:
-    installed: true
```
Comment (sed-i) [inline goss.yaml:6]: And now, given the new context, I agree with Dylan that we can keep this "rock linting" check in place.
We would remove the `package` section if we were to convert this from "internal" to "external" check.

**Review by sed-i — APPROVED** [review]

File: goss.yaml (hunk 1 above)
Comment (lucabello) [reply goss.yaml:6]: Got you, keeping that check then!

**Comment by lucabello** [comment]
Superseded by #51

**Pull request closed without merging**